pkg/providers/aws/provider_postgres.go

```diff
@@ -203,6 +202,11 @@ func (p *AWSPostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha
 		}
 		return nil, croType.StatusMessage(fmt.Sprintf("changes detected, modifyDBInstance() in progress, current aws rds resource status is %s", *foundInstance.DBInstanceStatus)), nil
 	}
+	// Add Tags to Aws Postgres resources
+	_, err = p.TagRDSPostgres(ctx, cr, rdsSvc, foundInstance)
```

Comment:
Just thinking, but shouldn't we be handling a status here? As this is the last step, it should return `creation successful` or else an error status.

Reply:
@ciaranRoche i'm not fully understanding your comment here, we handle the returned error state from this function call, on L212 we perform the last return step. are you talking about handling the first, currently ignored, value that's returned by the `#TagRDSPostgres` function?